org-asterion/__main__.py

```diff
@@ -87,17 +87,27 @@ def org_exists(self):
     user=new_user.name
 )
 
+# Obtain an aws access key and apply to the new user
+new_user_access_key = aws.iam.AccessKey(
+    "asterion-infra-aws-user-access-key",
+    user=new_user.name
+)
+
 # Export password for the user
 export("New user password", new_user_login.password)
 
+# Export access key for the user
+export("Encrypted secret access key", new_user_access_key.encrypted_secret)
```

Comment:
Curious, are these values random / dynamically generated when you create the key on line 91?